qqqwww123cdv/hometask-5/5.sql

```diff
@@ -0,0 +1,5 @@
+SELECT name, COUNT(film_id) AS number_of_films

```

Comment:
let's specify column names in front of fields if we have more than one table in our selection